app/controllers/catalog_controller.rb

```diff
@@ -1560,6 +1573,16 @@ def add_ansible_tower_job_template_vars(st)
       nil : ConfigurationScript.find_by(:id => @edit[:new][:template_id])
   end
 
+  def add_ansible_tower_container_template_vars(st)
+    st.options = {
+      :config_info => {
+        :provision => {
+          :container_template_id => @edit[:new][:template_id]
```

Comment:
This container_template_id is container template specific. How can it be shared with ansible tower? Or that will be added later?